WordPressAuthenticator/Unified Auth/View Related/Site Address/SiteAddressViewController.swift

```diff
@@ -468,7 +468,13 @@ private extension SiteAddressViewController {
             return
         }
         
-        WordPressAuthenticator.shared.delegate?.shouldPresentUsernamePasswordController(for: siteInfo, onCompletion: { (error, isSelfHosted) in
+        WordPressAuthenticator.shared.delegate?.shouldPresentUsernamePasswordController(for: siteInfo, onCompletion: { (error, isSelfHosted, injectedViewController) in
+            // Shortcircuit if clients provide a view controller to be inserted in the stack
+            if let customUI = injectedViewController {
```

Comment:
If client apps inject a view controller, push it to the current navigation controller. If not, fallback to the previous logic

This is why an enumeration, might make more sense here.